python/cudf/cudf/_lib/groupby.pyx

```diff
@@ -112,6 +113,7 @@ cdef class GroupBy:
         cdef Column col
         cdef Aggregation agg_obj
 
+        cdef bool scan = _is_all_scan_aggregate(aggregations)
```

Comment:
```suggestion
        cdef bool scan = _is_all_scan_aggregate(aggregations)
        if scan:
            # TODO: Once https://github.com/pandas-dev/pandas/issues/40695 is resolved, this will
            # have to also keep track of which requested aggregations are scans so that we can call
            # scan and aggregate accordingly.
            aggregations = {col_name: list(map(lambda agg_name: agg_name.replace('cum', ''), aggs))
                                       for col_name, aggs in aggregations.items()}
```

Reply:
Not all aggregations are strings. There may be functions too.